packages/components/src/Main/Main.tsx

```diff
@@ -1,110 +1,24 @@
-  params?: { [key: string]: string };
-  path?: string;
-  className?: string;
-  children?: React.ReactNode;
-}
-
-type FcalculateLocation = () => { staticPart: string[]; dynamic: undefined | { [key: string]: any } };
+export type MainProps = React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>;
+export type InternalMainProps = MainProps;
 
 /**
- * This is a component that wraps the page
+ * @deprecated Do not use this component, use HTML section tag
  */
-export const InternalMain: React.FunctionComponent<InternalMainProps> = ({ path, params = {}, children, className, ...props }) => {
-  const calculateLocation: FcalculateLocation = () => {
-    if (window?.insights?.chrome?.$internal?.store) {
-      const chromeState = window.insights.chrome.$internal.store.getState();
-      if (path && chromeState) {
-        return path.split('/').reduce(
-          (acc, curr) => {
-            if (curr.indexOf(':') === 0) {
-              acc.dynamic = {
-                ...acc.dynamic,
-                [`data-${toKebab(curr.substr(1))}`]: params[curr.substr(1)],
-              };
-            } else {
-              acc.staticPart = [...acc.staticPart, ...(curr !== '' ? [curr] : [])];
-            }
-
-            return acc;
-          },
-          { staticPart: [chromeState.chrome.appId], dynamic: {} }
-        );
-      }
-    }
-
-    return {
-      staticPart: [],
-      dynamic: undefined,
-    };
-  };
-
-  const { dynamic, staticPart } = calculateLocation();
+export const Main: React.FC<MainProps> = ({ children, className, ...props }) => {
+  useEffect(() => {
+    console.error(`Using deprecated "Main" component. Do not use it. Either remove it from your JSX or replace it by "section" HTML element.`);
+  }, []);
   return (
-    <DarkContext.Consumer>
-      {(theme = 'light') => {
-        const themeClasses = classNames({ [`pf-m-${theme}`]: theme === 'dark' });
-
-        return {
-          dark: (
-            <section
-              {...props}
-              {...dynamic}
-              page-type={staticPart.join('-')}
-              className={`${classNames(className, 'pf-l-page__main-section pf-c-page__main-section')} ${themeClasses}`}
-            >
-              {React.Children.map(children, (child) => {
-                return React.cloneElement(child as ReactElement, {
-                  className: 'pf-m-dark',
-                });
-              })}
-            </section>
-          ),
-          light: (
-            <section
-              {...props}
-              {...dynamic}
-              page-type={staticPart.join('-')}
-              className={`${classNames(className, 'pf-l-page__main-section pf-c-page__main-section')}`}
-            >
-              {children}
-            </section>
-          ),
-        }[theme];
-      }}
-    </DarkContext.Consumer>
+    <section {...props} className={`${classNames(className, 'pf-l-page__main-section pf-c-page__main-section')}`}>
```

Comment:
Maybe we should not deprecate this component. Testing the PR it looked like it provides some essential styling that some apps appear to rely on.